birdhouse/README.rst

```diff
@@ -114,6 +114,37 @@ Disk: at least 100 TB, depending how much data is hosted on Thredds and Geoserve
 In general, the more users, the more cpu cores and memory needed.  The more data, more memory and bigger and faster disks needed.
 
 
+Local deployment for development
+--------------------------------
+
+To deploy the birdhouse stack for development, a self-signed SSL certificate is required:
+
+First create a self signed certificate as described in the `LetsEncrypt documentation <https://letsencrypt.org/docs/certificates-for-localhost/#making-and-trusting-your-own-certificates>`_ .
+Then add the certificate you just generated to the trust store as described `here <https://ubuntu.com/server/docs/security-trust-store>`_ .
+Finally, combine the generated key and certificate files together to use as the `SSL_CERTIFICATE` for this stack.
+
+The following should work to generate a self-signed ssl certificate on ubuntu 20.04+:
+
+.. code-block::
+
+    openssl req -x509 -out localhost.crt -keyout localhost.key \
+      -newkey rsa:2048 -nodes -sha256 \
+      -subj '/CN=localhost' -extensions EXT -config <( \
+       printf "[dn]\nCN=localhost\n[req]\ndistinguished_name = dn\n[EXT]\nsubjectAltName=DNS:localhost\nkeyUsage=digitalSignature\nextendedKeyUsage=serverAuth")
+
+    # https://ubuntu.com/server/docs/security-trust-store
+    sudo cp localhost.crt /usr/local/share/ca-certificates/
```

Comment:
Oh I don't have this part in my Vagrant steps.  If you use Vagrant could you also update https://github.com/bird-house/birdhouse-deploy/blob/2b344d3383d8c0cd3f4cc4a0bf4266e3bae06a47/birdhouse/vagrant-utils/configure-pavics.sh#L65-L68

If not, it's okay, I can next time I try to rebuild my Vagrant box from scratch.  I do this about once a year.

Reply:
I don't know how the trust store works with the .crt, but on my side, I only did the concat of the files and it works. What is the purpose exactly of this step?